packages/netlify-cms-backend-bitbucket/src/git-lfs-client.ts

```diff
@@ -0,0 +1,183 @@
+import { filter, flow, fromPairs, map } from 'lodash/fp';
+import minimatch from 'minimatch';
+import { ApiRequest } from 'netlify-cms-lib-util';
+import { API as bitbucketAPI } from 'netlify-cms-backend-bitbucket';
```

Comment:
We shouldn't create this dependency by importing from the other backends

Reply:
I've updated the method